cellseg_gsontools/gradient.py

```diff
@@ -0,0 +1,101 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import shapely
+from helper import create_grid
+from matplotlib import cm
+
+
+def get_gradient(pdf):
+    """Convert distribution into gradient distribution.
+
+    Args:
+    ---------
+        pdf: 2D cell distribution
+
+    Returns:
+    ---------
+        np.ndarray: Shape (H, W) with gradient lengths
+    """
+    grad = np.gradient(np.array(pdf.T))
+
+    # Length for each vector
+    gradient_flat = np.sqrt(grad[0] ** 2 + grad[1] ** 2)
+
+    return gradient_flat
+
+
+def development_metric(pdf):
+    """Measures tissue differentiation based on cell distribution.
+
+    Args:
+    ---------
+        pdf: 2D cell distribution
+
+    Returns:
+    ---------
+        float: measure of development
+    """
+    return np.sum(np.abs(get_gradient(pdf)))
+
+
+def arrow_plot(z, points, dx, dy):
```

Comment:
Plotting functions could be moved to their own `plotting`/`plots` folder/module for better repo structure and self containment.